openapi/components/schemas/SubscriptionChange.yaml

```
@@ -81,6 +81,13 @@ properties:
                   format: double
                   description: Usage limit amount in the currency of the order.
                   minimum: 0.01
+        trialEndTime:
+          description: Free trial item date time.
```

suggestion: (I think is the end time of the trial)
```suggestion
          description: Date and time when the free trial ends.
```

or

```suggestion
          description: Date and time when the trial ends.
```

---

Thank you for the suggestions, fields got updated, please re-review.